docs/reference-guides/providers/howto.md

```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 
 # Create & import a cluster using CAPI providers
 
-This guide goes over the process of creating and importing CAPI clusters with a selection of the officially supported providers.
+This guide goes over the process of creating and importing CAPI clusters with a selection of the officially verified providers.
```

Comment:
Doing some nitpicking however why not use same wording everywhere?

Reply:
Thanks @furkatgofurov7. I agree the term must be the same to be clear. I just updated this.